spec/requests/api/v1/users/verifications_spec.rb

```diff
@@ -36,5 +36,14 @@
         end
       end
     end
+
+    context 'N+1', :n_plus_one do # rubocop:disable RSpec/ContextWording
```

Comment:
Have you checked that it's raising if it's n+1?

Reply:
Yes. I injected one n+1 query in code, and get this result in falling specs:
`Expected to make the same number of queries, but got:
         5 for N=2
         6 for N=3
       Unmatched query numbers by tables:
         users (SELECT): 3 != 4
`

Reply:
I add specs for all GET requests